sarpy/visualization/remap.py

```diff
@@ -112,12 +131,12 @@ def nrl(x, a=1, c=220):
     # have scipy.
     from scipy.stats import scoreatpercentile as prctile
 
-    x = abs(x)
+    x = abs(x)  # native method...
     xmin = np.min(x)
     p99 = prctile(x[np.isfinite(x)], 99)
     b = (255 - c) / np.log10((np.max(x) - xmin) / ((a * p99) - xmin))
 
-    out = np.zeros_like(x, np.uint8)
+    out = np.zeros_like(x, np.uint8)  # np.zeros()? jeez.
```

Comment:
looks like we're pre-allocating since we're manipulating portions of the linear regions and nonlinear regions differently.  Is there a better way to do this?

Reply:
I was really just getting cranky here. This is just a rarely used function call. Preallocating is totally correct. This is the only place in the code this pattern is followed, so it just seems weird.